.travis.yml

```diff
@@ -3,17 +3,25 @@ go:
   - 1.10.x
   - 1.11.x
 
+jobs:
+  # Only check vendoring on the latest version, as what's vendored differs
+  # between versions of go.
+  include:
+    - go: 1.16.x
+      env: CHECK_VENDOR=1
+
 sudo: false
 
 before_install:
   - mkdir --parents $GOPATH/src/golang.org/x
     && git clone --depth=1 https://go.googlesource.com/lint $GOPATH/src/golang.org/x/lint
-    && go get golang.org/x/lint/golint
-  - go get github.com/vbatts/git-validation
+    && GO111MODULE=off go get golang.org/x/lint/golint
+  - GO111MODULE=off go get github.com/vbatts/git-validation
```

Comment:
instead of turning it on/off manually, you can probably just move to a directory that doesn't have go.mod such as /tmp

Reply:
I thought this was slightly cleaner (not having to create a `tmp` dir, or assume `/tmp` is present), but happy to update if you prefer that.

Reply:
Isn't this variable going away completely in 1.17? :grimacing:

Reply:
If we remove the old go versions, we can probably use `go install` for these. I tried to keep this PR focused on just the switch to go mod; but have some branches for follow ups.

I added go1.16 to the matrix in this PR as there's differences in vendoring / module resolution / go.sum between go versions, and I wanted to prevent additional churn if I switched in a follow up.

Happy to make changes in this PR though, if that's preferred